src/components/React/Howto.js

```diff
@@ -29,117 +29,133 @@ export default function Howto(props) {
+              </p>
+              <img
+                src="assets/images/howto/ridc-opacity-map-guidie.gif"
+                alt="Instructional gif on applying a color texture"
+                width="80%"
+              ></img>
+            </div>
             <hr className="my-4" />
           </Row>
           <Row>
-            <p>
-              Lastly, use the ranged slices to cut through the dataset on three
-              different axes.
-            </p>
-            <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/slice.gif"
-              alt="Instructional gif on cutting through the dataset on an axis"
-              width="100%"
-            />
+            <div className="text-center">
+              <p>
+                Lastly, use the ranged slices to cut through the dataset on
+                three different axes.
+              </p>
+              <img
+                src="assets/images/howto/ridc-slices-guide.gif"
+                alt="Instructional gif on cutting through the dataset on an axis"
+                width="80%"
+              />
+            </div>
             <hr className="my-4" />
           </Row>
 
           <h2 id="enabling-vr" className="mb-4">
             Enable VR on Web Browser
           </h2>
           <Row>
-            <p>
-              At this moment, VR on web browsers is only supported on Mozilla
-              Firefox. In order to enable this option in your computer, please
-              follow these steps:
-            </p>
-            <ol>
-              <li className="mx-3">
-                Right Click onn your Desktop and select Nvidia Control Panel
-              </li>
-              <li className="mx-3">
-                In the NVIDIA panel window, select the "Program Settings" Tab
-              </li>
-              <li className="mx-3">
-                In the 'Select a program' drop list select the Mozilla Firefox
-                (If you dont find it, you will have to click on the 'add'
-                button, look for firefox.exe). In the 'Select the preferred
-                graphics processor for this program' drop list select 'High-
-                Performance NVIDIA processor'
-              </li>
-            </ol>
-            <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/webvrNvidia2.gif"
-              alt="Instructional gif on enabling graphics accelerator on Mozilla Firefox"
-              width="100%"
-            />
+            <div className="text-center">
+              <p>
+                At this moment, VR on web browsers is only supported on Mozilla
+                Firefox. In order to enable this option in your computer, please
+                follow these steps:
+              </p>
+              <ol>
+                <li className="mx-3">
+                  Right Click onn your Desktop and select Nvidia Control Panel
```

Comment:
Does this only work for windows? (is that realistically the only OS that can do this anyway?)

Reply:
Are there non-nvidia options?

Reply:
@mcmcgrath13 
The VR mode works on firefox. I havent tested it on other browsers. 
what about change the statement to :

"VR mode is fully tested on mozilla firefox,  Although, it is highly possible that other modern browsers support this functionality"

Nvidia is a must.

Reply:
Maybe just lay out the prerequisites in the first section of these instructions?  e.g. (foo, bar, or baz headset, nvidia gpu, windows, firefox definitely supported (others may or may not work, not tested))